src/components/App.js

```diff
@@ -26,32 +29,30 @@ const LanguageSwitcher = () => {
 Top level router - routing between an actual form or supporting screens.
  */
 const App = ({...props}) => {
+  const config = useContext(ConfigContext);
   const {
     form: {translationEnabled},
     noDebug = false,
   } = props;
+
+  const AppDisplayComponent = config?.displayComponents?.app ?? AppDisplay;
+
+  const languageSwitcher = translationEnabled ? <LanguageSwitcher /> : null;
```

Comment:
I am wondering what is the reason of passing the jsx as prop to AppDisplay, instead of the component (`<LanguageSwitcher />` vs `LanguageSwitcher`)? (also, I guess there will come a `LanguageSwitcherDisplay`?)

Because if we used just the component, we could also specify here which component should be used for the `ProgressIndicatorDisplay`?

Reply:
`LanguageSelectionDisplay` already exists, see https://open-formulieren.github.io/open-forms-sdk/?path=/story/composites-language-selection--display&globals=locale:nl;locales.nl:Nederlands;locales.en:English

We deliberately pass an element (component bound with props, in this case none) instead of a component for which the props need to be provided, so that the layout component just needs to deal with re-organizing the layout of what goes where instead of calling the nested components with the correct props (which would require us to provide a mechanism to access those props).